src/App.jsx

```diff
@@ -1,17 +1,11 @@
 // import { useState } from "react";
 import './App.css';
+import Header from './components/header/Header';
 
 function App() {
-  testFun();
-  const testFun = () => {
-    console.log('this is a test');
-  };
-
   return (
     <div>
```

Comment:
Do we need a div here? Maybe use a fragment until you have more code later?

Reply:
@robester0403 actually I am not sure how we will go about that yet, as we will all need to have a discussion about that as the components are built, but I will keep that in mind.